src/modules/airdrop/services/airdrop-service.ts

```diff
@@ -177,14 +177,13 @@ export class AirdropService {
     return this.walletRewardsRepository.findOne({ where: { walletAddress } });
   }
 
-  private async getCommunityRewards(walletAddress: string): Promise<string | undefined> {
-    const userWallet = await this.userWalletService.getUserWalletByAttributes({ walletAddress });
+  private async getCommunityRewards(userId?: number): Promise<string | undefined> {
+    if (!userId) return undefined;
```

Comment:
NIT: Hmm, this looks weird to me to return `undefined` if the argument is `undefined`. Wouldn't it be cleaner to make `userId` mandatory and call this method conditionally?

Reply:
👍